app/src/main/java/io/homeassistant/companion/android/widgets/StaticWidget.kt

```diff
@@ -79,9 +82,18 @@ class StaticWidget : AppWidgetProvider() {
             val entityId: String? = widget?.entityId
```

Comment:
Maybe we should put the rest of this method inside a null check for `widget`?  Then not everything will be nullable here and make the code easier

Reply:
Also makes sense. Fixed that.